### PR TITLE
chore(version): Bump version

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
### Fixes
- fix(api): GraphQLResponse.toString() include data & errors ([https://github.com/aws-amplify/amplify-flutter/pull/5079](https://github.com/aws-amplify/amplify-flutter/pull/5079))
- fix(core): improve amplify configure api error message ([https://github.com/aws-amplify/amplify-flutter/pull/5021](https://github.com/aws-amplify/amplify-flutter/pull/5021))
- fix(notifications): allow offline configuration
- fix: wait for in progress multi part uploads to cancel for `pause` and `cancel`

### Features
- feat: Support Amplify Gen 2 outputs ([https://github.com/aws-amplify/amplify-flutter/pull/5073](https://github.com/aws-amplify/amplify-flutter/pull/5073))

Updated-Components: libgit2dart, amplify_lints, Amplify Flutter, Amplify Dart, Amplify UI, DB Common, Secure Storage, AWS Common, Smithy, Worker Bee

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Equartey — DISMISSED** [review]

File: packages/analytics/amplify_analytics_pinpoint_dart/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.4.2
+
+### Fixes
+- fix(notifications): allow offline configuration
```
Comment (NikaHsn) [inline packages/analytics/amplify_analytics_pinpoint_dart/CHANGELOG.md:4]: I think we should remove this

**Review by tyllark — APPROVED** [review]

File: packages/auth/amplify_auth_cognito_dart/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 0.11.2
+
+### Fixes
+- fix(core): improve amplify configure api error message ([#5021](https://github.com/aws-amplify/amplify-flutter/pull/5021))
+
```
Comment (NikaHsn) [inline packages/auth/amplify_auth_cognito_dart/CHANGELOG.md:5]: I think we should remove this

File: packages/notifications/push/amplify_push_notifications/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## 2.2.0
+
```
Comment (NikaHsn) [inline packages/notifications/push/amplify_push_notifications/CHANGELOG.md:2]: should we add ?
```
### Fixes
- fix(notifications): allow offline configuration
```

File: packages/storage/amplify_storage_s3_dart/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+## 0.4.2
+
+### Fixes
+- fix(core): improve amplify configure api error message ([#5021](https://github.com/aws-amplify/amplify-flutter/pull/5021))
```
Comment (NikaHsn) [inline packages/storage/amplify_storage_s3_dart/CHANGELOG.md:4]: I think we should remove this

**Review by Equartey — APPROVED** [review]

**Review by NikaHsn — DISMISSED** [review]

**Review by Equartey — APPROVED** [review]

**Review by tyllark — APPROVED** [review]

**Pull request merged**